ext/opcache/jit/zend_jit_vm_helpers.c

```diff
@@ -176,9 +176,8 @@ ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL zend_jit_copy_extra_args_helper_no_skip_re
 	return zend_jit_copy_extra_args_helper_ex(ZEND_OPCODE_HANDLER_ARGS_PASSTHRU_EX false);
 }
 
-bool ZEND_FASTCALL zend_jit_deprecated_helper(OPLINE_D)
+bool ZEND_FASTCALL zend_jit_deprecated_helper(zend_execute_data *call)
 {
-	zend_execute_data *call = (zend_execute_data *) opline;
```

Comment:
I totally don't understand the details of this PR, but this change alone LGTM. It was super confusing to me when I implemented `#[\NoDiscard]`.